tests/Tests/500__jupyterhub/test-minimal-image.robot

```diff
@@ -11,6 +11,8 @@ Suite Teardown      End Web Test
 
 Force Tags          Smoke    Sanity    JupyterHub
 
+*** Variables ***
+${RequirementsFileRepo}=    https://github.com/redhat-rhods-qe/useful-files.git
```

Comment:
@pranavsuryaw @lugi0 Instead of creating a new repo for this, I think it would be cleaner to have the file here:

https://github.com/redhat-rhods-qe/ods-ci-notebooks-main/tree/main/notebooks/500__jupyterhub/test-minimal-image/requirements.txt